src/webui/www/private/scripts/prop-general.js

```diff
@@ -107,109 +107,109 @@ window.qBittorrent.PropGeneral = (function() {
 
                     const totalSize = (data.total_size >= 0) ? window.qBittorrent.Misc.friendlyUnit(data.total_size) : "";
-                    $("total_size").set("html", totalSize);
+                    $("total_size").textContent = totalSize;
 
                     const pieces = (data.pieces_num >= 0)
                         ? "QBT_TR(%1 x %2 (have %3))QBT_TR[CONTEXT=PropertiesWidget]"
                         .replace("%1", data.pieces_num)
                         .replace("%2", window.qBittorrent.Misc.friendlyUnit(data.piece_size))
                         .replace("%3", data.pieces_have)
                         : "";
-                    $("pieces").set("html", pieces);
+                    $("pieces").textContent = pieces;
 
-                    $("created_by").set("text", data.created_by);
+                    $("created_by").textContent = data.created_by;
 
                     const additionDate = (data.addition_date >= 0)
                         ? new Date(data.addition_date * 1000).toLocaleString()
                         : "QBT_TR(Unknown)QBT_TR[CONTEXT=HttpServer]";
-                    $("addition_date").set("html", additionDate);
+                    $("addition_date").textContent = additionDate;
 
                     const completionDate = (data.completion_date >= 0)
                         ? new Date(data.completion_date * 1000).toLocaleString()
                         : "";
-                    $("completion_date").set("html", completionDate);
+                    $("completion_date").textContent = completionDate;
 
                     const creationDate = (data.creation_date >= 0)
                         ? new Date(data.creation_date * 1000).toLocaleString()
                         : "";
-                    $("creation_date").set("html", creationDate);
+                    $("creation_date").textContent = creationDate;
 
                     const torrentHashV1 = (data.infohash_v1 !== "")
                         ? data.infohash_v1
                         : "QBT_TR(N/A)QBT_TR[CONTEXT=PropertiesWidget]";
-                    $("torrent_hash_v1").set("html", torrentHashV1);
+                    $("torrent_hash_v1").textContent = torrentHashV1;
 
                     const torrentHashV2 = (data.infohash_v2 !== "")
                         ? data.infohash_v2
                         : "QBT_TR(N/A)QBT_TR[CONTEXT=PropertiesWidget]";
-                    $("torrent_hash_v2").set("html", torrentHashV2);
+                    $("torrent_hash_v2").textContent = torrentHashV2;
 
-                    $("save_path").set("html", data.save_path);
+                    $("save_path").textContent = data.save_path;
 
-                    $("comment").set("html", window.qBittorrent.Misc.parseHtmlLinks(window.qBittorrent.Misc.escapeHtml(data.comment)));
+                    $("comment").textContent = window.qBittorrent.Misc.parseHtmlLinks(window.qBittorrent.Misc.escapeHtml(data.comment));
```

Comment:
In a situation like this one, I think the use of `textContent` will break the anchor element that's being injected by `parseHtmlLinks`.

Reply:
Yes, `textContent` sanitizes all markup, `parseHtmlLinks` output will be borked.

Reply:
Fixed, thanks!